cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,141 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-16976` - UI resets the default version of plugins for specific user during upgrade. When users upgrade from 6.1.2 to 6.1.3 or later UI will reset the default version of plugin the user has already chosen. Post upgrade if the user uses the same plugin UI will choose the latest version of the same plugin.
+
+- :cask-issue:`CDAP-17000` - Changed default value of spark.network.timeout to 10 minutes to make pipeline execution more stable for shuffle heavy pipelines.
+
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-17003` - Removed redundant validations from BQ sink , this should reduce calls to BigQuery.getTable()
+
+- :cask-issue:`CDAP-16530` - Fixed joiner output schema generation to be deterministic, using the same ordering as they had in the input data.
+
+- :cask-issue:`CDAP-17016` - Verified on the change in the delta replicator. Whenever a pipeline was created and ran, the target table created in BQ side will take the primary key as clustering columns as expected. 
+
+- :cask-issue:`CDAP-15869` - Preview displays logical types as strings.
+
+- :cask-issue:`CDAP-16222` - Fixed the package references in the dynamic spark plugin to use io.cdap instead of co.cask.
+
+- :cask-issue:`CDAP-16340` - Fixed the joiner plugin to allow a nullable key on one side and a non-nullable key on the other.
+
+- :cask-issue:`CDAP-16367` - Fixed a bug where field lineage is incorrect when a source is directly connected to a sink.
+
+- :cask-issue:`CDAP-16487` - Fixed regex for empty filter in Wrangler UI
+
+- :cask-issue:`CDAP-16731` - Fixed a bug that the groupBy aggregator requires a different alias for the field name.
+
+- :cask-issue:`CDAP-16747` - Modified loading screen for preview tab. 
+
+- :cask-issue:`CDAP-16760` - Fixed a bug where memory, cpu, and engine config properties were not being set for sparkprogram plugins
+
+- :cask-issue:`CDAP-16786` - Fixes listing pipelines by tags in Pipelines list UI.
+
+- :cask-issue:`CDAP-16797` - CDAP UI now validates post run actions before adding to pipeline in studio.
+
+- :cask-issue:`CDAP-16845` - Fixed a bug that started running preview for pipelines with post-run actions even if user chose option to not run preview. 
+
+- :cask-issue:`CDAP-16870` - Fixed PySpark support to work with Spark 2.1.3+
+
+- :cask-issue:`CDAP-16879` - 'Truncate table' and 'update schema' options if set together, will apply only WRITE_TRUNCATE to BQ job.
+
+- :cask-issue:`CDAP-16880` - Removed schema validation from BQ sink when 'truncate table' option is set. 
+
+- :cask-issue:`CDAP-16891` - Unsupported pipelines in drafts would be upgraded when users open them.
+
+- :cask-issue:`CDAP-16927` - Verified on the latest version of Azure Blob Store Plugin. If the account name is not ended with ".blob.core.windows.net", the validation will fail with error message to tell user to add that suffix.
```

Comment:
Added validation to ensure that account name ends with ".blob.core.windows.net" in the Azure Blob Store plugin

Reply:
fixed